src/test/java/ltd/clearsolutions/lingualeo/client/LingualeoClientIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+package ltd.clearsolutions.lingualeo.client;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class LingualeoClientIntegrationTest {
+
+    @Test
+    void auth_PutCorrectCredentials_shouldReturnMap() throws IOException {
+
+        LingualeoClient lingualeoClient = new LingualeoClient("vasyll.danylenko@gmail.com", "6485644Df");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        InputStream inputFile = getClass().getResourceAsStream("/AuthorizationResponse.json");
```

Comment:
Move this code to private method
String getActualResultFromFile(String fileName)

InputStream inputFile = getClass().getResourceAsStream("/AuthorizationResponse.json");
        BufferedReader readFile = null;
        if (inputFile != null) {
            readFile = new BufferedReader(new InputStreamReader(inputFile, StandardCharsets.UTF_8));
        }

        Map<String, Object> actual = objectMapper.readValue(readFile,
                new TypeReference<>() {
                });